### ISSUE TITLE
Add chrome extension auto reload

### ISSUE THREAD

**Issue by bobharley** [bobharley/facebook-md#8]
The development of this chrome extension is laborious. Every time you modify the styles you have to reload the extension and refresh the page to see the changes. It'll be a lot easier if the extension auto reloads whenever you modify the files.

**Comment by badanomaly** [comment]
You can use - https://chrome.google.com/webstore/detail/extensions-reloader/fimgfedafeadlieiabdeeaodndnlbhid

**Comment by bobharley** [comment]
@badanomaly thanks but I already tried that. I still have to click on the reload icon or visit the reload url. I'm looking for a way to integrate it to my gulpfile. Maybe there's a way to automate this I just don't know how. It should reload the extension every time a file has been changed.

**Comment by TechiePriyaRanjan** [comment]
You want a chrome extension which reloads the page, every time you make changes to it.. Right..✌️?

**Comment by bobharley** [comment]
@TechiePriyaRanjan no. I need something that realoads an unpacked chrome extension every time I make changes into it.
